src/utils/fetchTautulli.ts

```diff
@@ -90,23 +93,64 @@ export async function getLibraries(
   excludeInactive = true,
 ): Promise<TautulliLibrary[]> {
   const settings = getSettings()
+  const libraries = (await fetchTautulli<TautulliLibrary[]>('get_libraries'))
+    ?.response?.data
+  if (libraries == null) {
```

Comment:
Please add ampty line after const declaration for readability and consistent styling. I will try to add linting enforcement for this in the future.